.github/workflows/action.yml

```
@@ -0,0 +1,44 @@
+name: Specs
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+  pull_request_target:
+
+jobs:
+  test:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        ruby-version:
+          - "2.3"
+          - "2.4"
+          - "2.5"
+          - "2.6"
+          - "2.7"
+          - "3.0"
```

NIT: You can do it in one-line
```suggestion
        ruby-version: ['2.3', '2.4', '2.5', '2.6', '2.7', '3.0']
```